### ISSUE TITLE
importing doesn't work anymore

### ISSUE THREAD

**Issue by joachimvdnab** [WebReflection/url-search-params#34]
When importing as 
`import URLSearchParams from 'url-search-params'`
and using it further in the code as
`const urlParams = new URLSearchParams(window.location.search)`
I get the following error:
TypeError: _urlSearchParams2.default is not a constructor
This started after the version update from 0.10.0 -> 0.10.1

**Comment by WebReflection** [comment]
Please open a ticket to whatever bundler you are using:
https://github.com/WebReflection/url-search-params/blob/master/package.json#L22

The package.json clearly state which CJS module you should import, and there is no ESM (or `module`) entry there so it's a problem of your bundler.

**Comment by mieszko4** [comment]
How it is not a breaking change?

**Comment by WebReflection** [comment]
> How it is not a breaking change?

It's very simple, nothing broke here, or in travis, and no API changes whatsoever were made.

I've made the module usable through unpkg CDN and that relvealed a bug of **your** bundler.

However, I've reverted the change in `0.10.2`, but since this is an absurdity I've put the change back in `1.0.0` so now it's officially a breaking change, and bundlers need to fix their issue.
